internal/migration/extract_migration_test.go

```diff
@@ -0,0 +1,67 @@
+package migration
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/alecthomas/assert/v2"
+
+	"github.com/TBD54566975/ftl/internal/schema"
+	"github.com/TBD54566975/ftl/internal/sha256"
+)
+
+func TestExtractMigrations(t *testing.T) {
+	t.Run("Valid migrations", func(t *testing.T) {
+		// Setup
+		migrationsDir := t.TempDir()
+		targetDir := t.TempDir()
+
+		// Create dummy migration files
+		migrationSubDir := filepath.Join(migrationsDir, "testdb")
```

Comment:
Didn't want to use `testdata` for the fixtures?

Reply:
I thought that was just for integration tests? Also it's possible Chad may have written this test...

Reply:
Hahaha 😂 

Nope `testdata` is standard Go for any test that has fixtures.